src/website/webview/templates/draft_list.html

```diff
@@ -0,0 +1,102 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h1>Drafts</h1>
+<div id="suggestion-list">
+  {% for object in object_list %}
+    <article class="suggestion">
+      <div class="row1">
+        <a class="cve-id" href="https://nvd.nist.gov/vuln/detail/{{ object.cve.cve_id | urlencode }}">
+          {{ object.cve.cve_id }}
+        </a>
+        <div class="severity {{ object.base_severity }}">
+          {% if object.base_severity == "LOW" %}
+            Low
+          {% elif object.base_severity == "MEDIUM" %}
+            Medium
+          {% elif object.base_severity == "HIGH" %}
+            High
+          {% elif object.base_severity == "CRITICAL" %}
+            Critical
+          {% elif object.base_severity == "NONE" %}
+            No severity data
+          {% else %}
+            Unknown Severity
+          {% endif %}
+        </div>
+      </div>
+      <h2>{{ object.package_name }}</h2>
+      <!-- XXX: just another idea how to do it -->
+      <details class="description">
```

Comment:
Yeah I don't really agree with this. Semantically I don't think that the title is just a summary of the description and the unfolding element makes the layout more janky and wastes space on mobile.

![tmp Y9Ji6RTXal](https://github.com/user-attachments/assets/2ab65892-3c0a-4db0-8fe1-f5446a6e0fe0)

Reply:
Reading a bunch of them, it does look like the title is a reasonable summary of the more detailed description, this is why I tried this out in the first place. We can figure out the word wrapping. The details tag has two advantages:
- no JS needed
- doesn't jump around when toggling state. the `show more` thing appears somewhere else depending on collapsed state

Reply:
Here's an alternative rendering:

![image](https://github.com/user-attachments/assets/270d57a8-55bf-4780-8ba9-69ee94123570)

Reply:
What actually can't be done without JS is moving the "Show less..." after the contents and still keep it interactive. But again, that would mean things would jump around.

I really don't care much whether it's the classic triangle or a word, but I slightly prefer the triangle because it has well-established semantics that don't require the user to parse text.

Reply:
Here's what it's like with a triange:

![image](https://github.com/user-attachments/assets/7ea24645-c18d-4a60-8f89-84b686e22011)

Reply:
deal!